src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -45,7 +46,7 @@ const BottomBox = ({
 
 const OutlookDots = ({ outlook, ariaLabel }: { outlook: number; ariaLabel: string }) => (
   <div role="figure" className="flex flex-row gap-2" aria-label={ariaLabel}>
-    {Array.from({ length: outlook }).map((_, idx) => (
+    {Array.from({ length: 5 }).map((_, idx) => (
```

Comment:
Fixed a bug. Dots should always be visible, only the the coloring changes based on the value.